C_CPP/break_continue_in_c_cpp.md

```diff
@@ -0,0 +1,51 @@
+# Break and Continue in C/CPP
+---
+Break and Countiue are *keywords* used in C/CPP.
+## Break
+* There are situtations where we want to jump out of loop instantly, no matter what is written after that.
```

Comment:
instead of "jump out of loop instantly" use "terminate

Reply:
I have made the changes as mentioned . Kindly check thanku